specification/contributors.md

```diff
@@ -1,3 +1,36 @@
+## Steering Committee Members
+
+Thanks to the following FOCUS Steering Committee Members for their contributions to this specification.
+
+* Anne Johnston (Capital One)
+* Michael Flanakin (Microsoft)
+* Mike Fuller (FinOps Foundation)
+* Roy Wolman (Amazon Web Services)
+* Sarah McMullin (Google)
+* Tim O'Brien (Walmart)
+* Udam Dewaraja (FinOps Foundation)
+
+## Maintainers
+
+Thanks to the following FOCUS Maintainers for their contributions to this specification.
+
+* Anne Johnston (Capital One)
+* Christopher Harris (Datadog)
+* Dennis Park-Rodriguez (Amazon Web Services)
+* Erik Peterzon (CloudZero)
+* Irena Jurica (Neos)
+* Joaquin Prado (FinOps Foundation)
+* Karl Kraft (Walmart)
+* Larry Advey (Twilio)
+* Michael Flanakin (Microsoft)
+* Mike Fuller (FinOps Foundation)
+* Riley Jenkins (DOMO)
+* Roy Wolman (Amazon Web Services)
+* Rupa Patel (Google)
+* Sarah McMullin (Google)
+* Tim O'Brien (Walmart)
+* Udam Dewaraja (FinOps Foundation)
+
```

Comment:
Seems like we would want to at least not double count members between maintainers and members

Reply:
Was thinking the same but don't know a good way to manifest this other than 1 list like:

```
John Doe
Jane Doe [Maintainer]
Mike Doe [Steering Committee]
Alex Doe [Maintainer, Steering Committee]
```

Reply:
Will just remove Maintainers from Member section for now.